README.md

```diff
@@ -82,6 +85,22 @@ It can:
 
 - Revoke any timelocked action except it cannot revoke a pending fee.
 
+### Idle Supply
+
+In some cases, the vault's curator or allocators may want to keep some funds "idle", to guarantee lenders some extent of liquidity from the vault (beyond the liquidity of each of the vault's markets).
+
+To achieve this, it is advised to allocate "idle" funds to any market on Morpho Blue having:
+
+- The vault's asset as loan token.
+- No collateral token (`address(0)`).
+- An arbitrary IRM (`address(0)` to save gas).
```

Comment:
will `address(0)` be allowed as IRM ?

Reply:
Being discussed, this PR cannot be merged as is indeed